libusb/core.c

```diff
@@ -1272,10 +1272,14 @@ int API_EXPORTED libusb_get_max_alt_packet_size(libusb_device *dev,
 DEFAULT_VISIBILITY
 libusb_device * LIBUSB_CALL libusb_ref_device(libusb_device *dev)
 {
+#ifdef NDEBUG
+	(void) usbi_atomic_inc(&dev->refcnt);
+#else
 	long refcnt;
 
 	refcnt = usbi_atomic_inc(&dev->refcnt);
 	assert(refcnt >= 2);
```

Comment:
Instead, you can just unconditionally add:

```
(void)refcnt;
```

after the assert.

Reply:
Oops, indeed! I obviously had too little coffee this morning 🤦‍♂️ Fixed 😊